go/src/koding/klient/machine/index/change.go

```diff
@@ -103,6 +103,25 @@ func (cm *ChangeMeta) Coalesce(newer ChangeMeta) ChangeMeta {
 	}
 }
 
+// String implements fmt.Stringer interface and pretty prints stored change.
+func (cm *ChangeMeta) String() string {
+	cpy := atomic.LoadUint64((*uint64)(cm))
+
+	const cmstr = "LRaduH"
```

Comment:
this is a solution used in [std's os.FileMode.String()](https://golang.org/src/os/types.go?s=2562:2595#L49)

Adding new lines between iota const will break the tests. I can make it a map(however tried to avoid this). wdyt